contributors/devel/api_changes.md

```diff
@@ -462,8 +462,26 @@ generator to create it from scratch.
 Unsurprisingly, adding manually written conversion also requires you to add
 tests to `pkg/api/<version>/conversion_test.go`.
 
+## Generate Code 
 
-## Generate protobuf objects
+When any `types.go` change is made, the generated code must be updated
+by running the generators. There are many small generators that
+run. They have some dependencies on each other so they must be run in
+a certain order.
+
+Approximately:
+ - `defaulter-gen`
+ - `deepcopy-gen`
+ - `conversion-gen`
+ - `client-gen`
+ - `lister-gen` which depends upon the output of `client-gen`.
+ - `informer-gen` which depends upon the output of `lister-gen` and `client-gen`.
+ - `codecgen` for fast json serialization with a codec.
+
+Many of the generators are based on
```

Comment:
I don't think the flags are treated that same for each of the generators in all cases.  :(